features/upgrade/routing/upgrade.feature

```diff
@@ -208,3 +208,85 @@ Feature: Routing and DNS related scenarios
     Then the step should succeed
     And the output should contain "Hello-OpenShift web-server-rc"
     """
+
+
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
```

Comment:
I think normal user can idle/wakeup the service, `admin` user is not required.

Reply:
done

Reply:
so `@admin` tag can be removed from this case.